modules/outbound-domain-event-handler/src/shared/config.ts

```diff
@@ -43,6 +47,20 @@ const config = Convict({
         default: 'info',
         env: 'LOG_LEVEL',
     },
+    API_SERVER: {
+        ENABLED: {
+            doc: 'Whether to enable API server or not',
+            format: 'Boolean',
+            default: false,
+            env: 'API_SERVER_ENABLED',
```

Comment:
I think we need to make these specific to the handler now, perhaps something like this:
```suggestion
            env: 'DOMAIN_EVENT_API_SERVER_ENABLED',
```

Note we should go back and update the Command handler to match the same as above, except using something like a `COMMAND_EVENT_*` prefix

Reply:
Alternative perhaps the prefixes should be simplified to something like:
- `SDK_ODH_*` <-- ODH = outbound domain handler
- `SDK_OCH_*` <-- OCH = outbound command handler

etc.

This would only be applicable for configs that are specific to the Service/Handler, and where there is commonality, it could stay as is --- unless we can think of a reason not too do so?

Reply:
Since the configs are entirely seperate no prefix should should be fine for all configs. We haven't added prefix for any other configs.

Or is there a case that we'd have both handlers running from the same enviorment, then we'd have to prefix everything.

Reply:
> Or is there a case that we'd have both handlers running from the same enviorment, then we'd have to prefix everything.

We do, `yarn start` when we run the Integration Tests.